averbis/core/_rest_client.py

```diff
@@ -1026,8 +1026,10 @@ def _build_url(self, endpoint):
 
     def __request(self, method: str, endpoint: str, **kwargs) -> dict:
         raw_response = self._run_request(method, endpoint, **kwargs)
-        response = raw_response.json()
-        self.__handle_error(response)
+        if raw_response.ok:
+            response = raw_response.json()
```

Comment:
There can be cases where the result is ok but the message is not JSON - then we again get a confusing error message.

Reply:
Thanks. Dealt with the issue in the latest commit.